ingress-inquisition/main.go

```diff
@@ -28,12 +28,12 @@ func main() {
 	securityGroupRuleDetails, err := utils.FindUnusedSecurityGroupRules(ctx, ec2Client, securityHubClient)
 	if err != nil {
 		log.Fatalf("Error finding unused security group rules: %v", err)
-	}
-
-	fmt.Println("\n ")
+	} else if len(securityGroupRuleDetails) == 0 {
+		fmt.Println("No unused security groups found")
+	} else if args.Execute {
```

Comment:
In a future PR, everything in this block should be encapsulated in one function.